stl/inc/xutility

```diff
@@ -4693,10 +4681,20 @@ _NODISCARD _Iter_diff_t<_FwdIt> count(
     _ExPo&& _Exec, const _FwdIt _First, const _FwdIt _Last, const _Ty& _Val) noexcept; // terminates
 #endif // _HAS_CXX17
 
-// FUNCTION TEMPLATE _Count_pr
+// FUNCTION TEMPLATE is_permutation
 template <class _InIt, class _Ty, class _Pr>
-_Iter_diff_t<_InIt> _Count_pr(_InIt _First, _InIt _Last, const _Ty& _Val, _Pr _Pred) {
-    // count elements that match _Val, using _Pred
+_NODISCARD _InIt _Find_pr(_InIt _First, const _InIt _Last, const _Ty& _Val, _Pr _Pred) {

```

Comment:
Could that be `constexpr` given that it essentially is a derivate of `std::find_if`

Reply:
Yes it can, and indeed #425 makes that change :)